pandas/io/sas/sas_xport.py

```diff
@@ -279,6 +279,10 @@ def _read_header(self):
         # read file header
         line1 = self._get_row()
         if line1 != _correct_line1:
+            if "**COMPRESSED**" in line1:
```

Comment:
Is this value in any file format document provided by SAS?  If so,  would be good to include a reference.

Reply:
OK, I've added it in a new commit